### PR TITLE
Status bar left tile

### PR DESCRIPTION
**Pull request by iam4x**
Add an indication on status bar when linter didn't receive errors:

screen shot 2015-05-26 at 13 07 36

screen shot 2015-05-26 at 13 07 43

### CODE REVIEW

**Comment by steelbrain** [comment]
Beautiful, I am gonna test it locally as soon as I'm on ubuntu again

File: lib/linter.coffee [hunk 1]
```diff
@@ -96,6 +97,9 @@ class Linter
       @progress onChange, false
 
   render: ->
+    # Update `LeftTile` of `StatusBar`
+    @View.updateLeftTile(@Messages.length)
```
Comment (steelbrain) [inline lib/linter.coffee]: How about we drop these `()` :P

File: lib/view-left-tile.coffee [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{View} = require 'space-pen'
+
+module.exports = class LeftTile extends View
+  @content: (nbErrors) ->
+    if nbErrors
+      return @div class: 'linter-error inline-block', =>
+        @span class: 'badge badge-flexible badge-error', =>
+          @text '✘'
+        @text " #{nbErrors} lint errors"
```
Comment (steelbrain) [inline lib/view-left-tile.coffee]: How about `errors` instead of `lint errors`

File: lib/view-left-tile.coffee [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    else
+      return @div class: 'linter-success inline-block', =>
+        @span class: 'badge badge-flexible badge-success', =>
+          @text '✔'
+        @text ' No lint error'
```
Comment (steelbrain) [inline lib/view-left-tile.coffee]: `no errors` maybe?

**Comment by steelbrain** [comment]
I get the generic unknown character (the box like thingy) for both the x and the tick. I am on Ubuntu 15.04  One Dark UI Theme and Atom Dark Syntax Theme. without any custom styles that would override the font.
screenshot from 2015-05-26 05-41-54

**Comment by iam4x** [comment]
screen shot 2015-05-26 at 15 09 42
screen shot 2015-05-26 at 15 09 48

**Comment by steelbrain** [comment]
Looks Good to Me.

**Pull request merged**